src/library/scala/collection/View.scala

```diff
@@ -211,6 +211,12 @@ object View extends IterableFactory[View] {
     def iterator = underlying.iterator.flatMap(f)
   }
 
+  /** A view that collects elements of the underlying collection. */
+  @SerialVersionUID(3L)
+  class Collect[+A, B](underlying: SomeIterableOps[A], pf: PartialFunction[A, B]) extends AbstractView[B] {
```

Comment:
not final?

Reply:
None of those classes are. More specific View types like IndexedSeqView have to be able to extend them.